components/LinkGrid/src/index.scss

```diff
@@ -0,0 +1,183 @@
+.denhaag-link-grid {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  overflow-x: hidden;
+  background-color: var(--denhaag-link-grid-panel-background-color);
```

Comment:
De benaming van variabelen doen we zoals: 
```
.denhaag-link-grid {
background-color: var(--denhaag-link-grid-background-color);
}
```
Het gedeelte `--denhaag-link-grid` de selectornaam, en de propertynaam daarna `-background-color`
(ik weet dat dit gedoe is, maar zo houden we de variabelen leesbaarder en consistent)

Reply:
Done in de ge-update PR!